test/plugin/secretless_test_harness/README.md

```diff
@@ -0,0 +1,17 @@
+# secretless TCP plugin test harness
+
+1. Create your own plugin shared object file with the filename `example.so` in this folder. Otherwise, you can run `make build_example` to build the example.
+2. In a separate shell run `make run_echo` to run the TCP echo server at 127.0.0.1:6174.
+3. In a separate shell run `make run_proxy` to run the test harness with the TCP plugin loaded.
```

Comment:
This should be secretless itself with some mix of internal funcs rather than a 1-off codebase